internal/manifests/queryfrontend/query_frontend.go

```diff
@@ -246,6 +255,33 @@ func deployment(params manifestutils.Params) (*v1.Deployment, error) {
 	return d, nil
 }
 
+func enableMonitoringTab(tempo v1alpha1.TempoStack, jaegerQueryContainer corev1.Container) (corev1.Container, error) {
+	// TODO (pavolloffay) disable/enable monitoring tab https://github.com/grafana/tempo-operator/issues/464
+	container := corev1.Container{
+		Env: []corev1.EnvVar{
+			{
+				Name:  "METRICS_STORAGE_TYPE",
+				Value: "prometheus",
+			},
+			{
+				Name:  "PROMETHEUS_SERVER_URL",
+				Value: tempo.Spec.Template.QueryFrontend.JaegerQuery.MonitorTab.PrometheusEndpoint,
+			},
+		},
+		Args: []string{
```

Comment:
I need to add a condition here to set the TLS/token file only when deployed on OCP